src/github.com/stellar/horizon/resource/effects/main.go

```diff
@@ -161,18 +174,21 @@ type SignerCreated struct {
 	Base
 	Weight    int32  `json:"weight"`
 	PublicKey string `json:"public_key"`
+	Key       string `json:"key"`
 }
 
 type SignerRemoved struct {
 	Base
 	Weight    int32  `json:"weight"`
 	PublicKey string `json:"public_key"`
+	Key       string `json:"key"`
 }
 
 type SignerUpdated struct {
 	Base
 	Weight    int32  `json:"weight"`
 	PublicKey string `json:"public_key"`
+	Key       string `json:"key"`
```

Comment:
I made a judgement call...  it's not clear how often effects like this will be used in applications, so rather than bulking the resource up now I'll wait until someone has a genuine need.